src/containers/search/index.js

```diff
@@ -41,6 +41,11 @@ export default class SearchProvider extends Component {
     this.search(searchText);
   };
 
+  componentWillReceiveProps(prevProps, prevState) {
+    //searching when receiving props it helps to retain the searched data in table
+    this.search(this.state.searchText);
```

Comment:
try to remove component will receive props and replace with maybe component did update will work